openapi-spec/swagger.yaml

```diff
@@ -2328,6 +2445,13 @@ components:
           Description: The Days element is required for regular restore. Required for restoring archived object.
           example: 1
   requestBodies:
+    AKSK:
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/AKSK'
```

Comment:
Not sure if my understanding is right, but schema should have AK SK right? 
![image](https://user-images.githubusercontent.com/48085413/123683864-59d20080-d86a-11eb-9c90-c6d1aa5b2419.png)

Reply:
![image](https://user-images.githubusercontent.com/65751939/123685456-58a1d300-d86c-11eb-850f-4d8bb1bf515d.png)
Fixed